lambdas/data-migration2/tests/test-index.js

```diff
@@ -68,49 +71,34 @@ test.before(async (t) => {
   const { knex, knexAdmin } = await generateLocalTestDb(testDbName, migrationDir);
   t.context.knex = knex;
   t.context.knexAdmin = knexAdmin;
-});
 
-test.beforeEach(async (t) => {
   t.context.pdrPgModel = new PdrPgModel();
+  t.context.collectionPgModel = new CollectionPgModel();
+  t.context.providerPgModel = new ProviderPgModel();
+  t.context.executionPgModel = new ExecutionPgModel();
+  t.context.granulePgModel = new GranulePgModel();
+});
 
-  const collectionPgModel = new CollectionPgModel();
+test.beforeEach(async (t) => {
   const testCollection = fakeCollectionRecordFactory();
-  await collectionPgModel.create(
+  [t.context.collectionCumulusId] = await t.context.collectionPgModel.create(
     t.context.knex,
     testCollection
   );
   t.context.testCollection = testCollection;
 
-  const providerPgModel = new ProviderPgModel();
   const testProvider = fakeProviderRecordFactory();
-
-  await providerPgModel.create(
+  await t.context.providerPgModel.create(
     t.context.knex,
     testProvider
   );
   t.context.testProvider = testProvider;
 
-  const executionPgModel = new ExecutionPgModel();
   const executionUrl = cryptoRandomString({ length: 5 });
   t.context.executionUrl = executionUrl;
-
-  const testExecution = fakeExecutionRecordFactory({
-    url: executionUrl,
-  });
-  await executionPgModel.create(
-    t.context.knex,
-    testExecution
-  );
-  t.context.testExecution = testExecution;
 });
 
 test.after.always(async (t) => {
-  await t.context.granulesModel.deleteTable();
```

Comment:
With the preface that I'm probably  misunderstanding something - I think it's fine to have individual record teardowns in the test, but what's the reason for not also removing the dynamo table entirely to ensure that a teardown/kill -9/ctrl c mash doesn't foul up the dynamo env in localstack?

Reply:
I think I read these wrong as deleting PG tables, not dynamo tables. I can restore them

Reply:
8dabf1d2f23bb1e0f95dbee0a761a2cf1eb979d6